Person_rb

```diff
@@ -1,41 +1,27 @@
-# frozen_string_literal: true
-
 require_relative 'nameable'
-require_relative 'capitalize'
-require_relative 'trimcase'
-# class for library
-class Person < Nameable
-  # constructor method
-  attr_reader :id, :name, :age
+class Person < NameAble
+  attr_accessor :name, :parent_permission, :age, :classroom, :rentals
```

Comment:
- [ ] I don't think `:classroom` has any relationship with the person class, and it is not defined anywhere in the class either. Let's remove unnecessary code